src/app/dashboard/Missions/DetailsView.jsx

```diff
@@ -0,0 +1,246 @@
+import { Box, Container, Grid, Paper } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import AccessTimeIcon from "@material-ui/icons/AccessTime";
+import ArrowBackIcon from "@material-ui/icons/ArrowBack";
+import AttachMoneyIcon from "@material-ui/icons/AttachMoney";
+import LocationOnIcon from "@material-ui/icons/LocationOn";
+import PanToolIcon from "@material-ui/icons/PanTool";
+import PersonIcon from "@material-ui/icons/Person";
```

Comment:
For future reference you should be able to import all of from just `@material-ui/icons`